package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "foodify_capstone_project",
+    "version": "1.0.0",
+    "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+    "main": "index.js",
+    "scripts": {
+      "test": "jest",
+      "watch": "jest --watchAll",
+      "start": "webpack-dev-server --config webpack.config.js --open",
+      "build": "webpack"
+    },
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/ABDULALI3468/Foodify_capstone_project.git"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "bugs": {
+      "url": "https://github.com/ABDULALI3468/Foodify_capstone_project/issues"
+    },
+    "homepage": "https://github.com/ABDULALI3468/Foodify_capstone_project#readme",
+    "devDependencies": {
+      "@babel/core": "^7.18.5",
+      "@babel/plugin-transform-modules-commonjs": "^7.17.9",
+      "@babel/preset-env": "^7.18.2",
+      "babel-eslint": "^10.1.0",
+      "css-loader": "^6.7.1",
+      "eslint": "^7.32.0",
+      "eslint-config-airbnb-base": "^14.2.1",
+      "eslint-plugin-import": "^2.26.0",
+      "hint": "^2.0.0",
+      "html-webpack-plugin": "^5.5.0",
+      "isomorphic-fetch": "^3.0.0",
+      "jest": "^28.1.1",
+      "jest-environment-jsdom": "^28.1.1",
+      "style-loader": "^3.3.1",
+      "stylelint": "^13.13.1",
+      "stylelint-config-standard": "^21.0.0",
+      "stylelint-csstree-validator": "^1.9.0",
+      "stylelint-scss": "^3.21.0",
+      "webpack": "^5.72.1",
+      "webpack-cli": "^4.9.2",
+      "webpack-dev-server": "^4.9.1"
+    },
+    "dependencies": {
+      "npm": "^8.12.1"
+    },
+    "directories": {
+      "doc": "docs"
+    }
```

Comment:
- [ ] Please change the value of the testenvironment to jsdom by adding this block of code to `package.json`
```
"jest":{
    "testEnvironment": "jsdom"
}
```
When I run the test I receive failed cases as seen below. You can read more about it [here](https://stackoverflow.com/questions/69227566/consider-using-the-jsdom-test-environment)
![image](https://user-images.githubusercontent.com/45460192/175560617-846d100d-c0cf-42db-8018-1ee337d5ed35.png)